server/CMakeLists.txt

```diff
@@ -91,6 +91,28 @@ install(
   NAMELINK_COMPONENT ${IRODS_PACKAGE_COMPONENT_DEVELOPMENT_NAME}
 )
 
+# Initialize IRODS_CONFIGURATION_SCHEMA_PREFIX used by configuration schemas.
+# This variable helps to produce the correct URI for the "$id" JSON property.
+if (CMAKE_INSTALL_PREFIX STREQUAL "/")
```

Comment:
I think this case should also apply if `IRODS_HOME_DIRECTORY` is absolute.

Reply:
You're talking about the case where someone defines a value for `IRODS_HOME_DIRECTORY` on the command line, right?

Reply:
> You're talking about the case where someone defines a value for `IRODS_HOME_DIRECTORY` on the command line, right?

Correct.

> Should the handling of `IRODS_HOME_DIRECTORY` be done here or is it unrelated to non-package installs?

I'm not sure I understand the question. If `IRODS_HOME_DIRECTORY` is absolute, we don't want to prepend `CMAKE_INSTALL_PREFIX` at all in the schemas.

Reply:
I see.

Should anyone ever set `IRODS_HOME_DIRECTORY`?
Can you think of cases where someone may want to set that variable?

Reply:
Pretty much any case in which it is undesirable to have the home directory in `/var`

Reply:
Done.